services/QuillLMS/app/models/english_text.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: english_texts
+#
+#  id         :bigint           not null, primary key
+#  text       :text             not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+class EnglishText < ApplicationRecord
+  has_many :translated_texts
+  has_many :translation_mappings
+
+  STANDARD_COMMENT = "
```

Comment:
I think you want to use [Squiggly heredoc](https://ruby-doc.org/core-2.5.0/doc/syntax/literals_rdoc.html#label-Here+Documents) here to extract remove whitespace and indentation.., e.g.

```ruby
STANDARD_COMMENT = <<~STRING
     We are translating the instructions for an English-language grammar activity. The content of the activity itself 
     is not translated. Therefore, please leave words that sound like they are part of the activity in the original 
     english. Often they will between an HTML tag such as in <em>english word</em> or <ul>english  word</ul>.
STRING